modules/programs/eza.nix

```
@@ -0,0 +1,67 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  meta.maintainers = [ pkgs.lib.maintainers.cafkafk ];
```

as you have put `with lib;` the expression can be shortened to:
```suggestion
  meta.maintainers = [ pkgs.lib.maintainers.cafkafk ];
  meta.maintainers = [ maintainers.cafkafk ];
```